vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md

```diff
@@ -4,90 +4,85 @@ redirectFrom:/next/docs/ecr/ecr-setup-private-git.html
 # Install Bundles from a Private Git Repository
 
 ## Overview
-The standard deployment of the Entando Component Repository assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication in order to access content inside the Git repository.
+The standard deployment of the Entando Component Repository (ECR) assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication, but if a customer chooses to limit repository access to specific Git accounts a private key is required. This document details how to allow the ECR to access a private Git repository.
 
-A customer may however choose to limit repository access to specific Git user accounts. This document will
-describe the steps that can be followed to allow the ECR to log onto the Git server when pulling content from a
-Git repository.
-
-For more background information, please consult the folowing documentation:
+For more background information please consult
 1. [Entando custom resource reference](../../docs/concepts/custom-resources.md)
 2. [Entando Component Repository overview](../../docs/ecr/ecr-overview.md)
 
 ## Using SSH keys with Git
-Entando supports SSH keys to connect to Git repositories. Following the SSH approach, a developer that 
-connects to Git would typically generate a public/private keypair. The developer would then register the public
-key with the Git server, whilst storing the private key securely in the local filesystem. When the Git command line
-utility executes an operation that requires an authenticated user, it will automatically look for the private
-key locally and match it with the public key provided by the server. If the two match up, then the operation is allowed to complete.
+Entando supports connecting to Git repositories with SSH keys. While SSH Git URLs are accepted, HTTPS git URLs are not. For the private key to work correctly the SSH syntax of `git@github.com:THE-REPO-OWNER-NAME/THE-REPO-NAME.git` must be used (the HTTPS syntax of `https://github.com/THE-REPO-OWNER-NAME/THE-REPO-NAME.git` is not permitted).
```

Comment:
"For the private key to work correctly, the SSH syntax of `git@github.com:THE-REPO-OWNER-NAME/THE-REPO-NAME.git` must be used.."

sorry another comma needed.